apple/utils.bzl

```diff
@@ -165,7 +165,7 @@ def group_files_by_directory(files, extensions, attr):
                 if container in grouped_files:
                     grouped_files[container] += [f]
```

Comment:
can you also change this to be `grouped_files[container].append(f)` ? it'll be clearer it's a list being modified

Reply:
done